src/util-file.c

```diff
@@ -149,6 +155,25 @@ void FileForceTrackingEnable(void)
     g_file_force_tracking = 1;
 }
 
+void FileWriteMetaDisable(void)
```

Comment:
it seems all the log-filestore.c settings are globals. Might be nice to fix it but we can keep things as globals for now. I would like to see it move into log-filestore.c thought as it only has an effect there.